code/modules/mob/living/simple_animal/hostile/ordeal/white.dm

```diff
@@ -403,12 +404,12 @@
 /mob/living/simple_animal/hostile/ordeal/red_fixer/OpenFire()
 	if(busy)
 		return
-	if(prob(50) && (get_dist(src, target) < multislash_range) && (multislash_cooldown < world.time))
-		MultiSlash(target)
-		return
 	if(prob(80) && (beam_cooldown < world.time))
 		LaserBeam(target)
 		return
+	if(prob(50) && (get_dist(src, target) < multislash_range) && (multislash_cooldown < world.time))
+		MultiSlash(target)
+		return
```

Comment:
I like the change of making the beam more common than the slash by raising it's priority, having fought it I found the slash was very hard to punish AT ALL due to the rapid flick between being in front and behind you.